aws/data_source_aws_ecr_authorization_token_test.go

```diff
@@ -0,0 +1,60 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccAWSEcrDataSource_ecrAuthorizationToken(t *testing.T) {
```

Comment:
Please rename to `TestAccAWSEcrAuthorizationTokenDataSource_basic`

Reply:
fixed in 7bf9644d75ed3420a831aaff8bb98d053a2a0e26